docs/links.ts

```diff
@@ -122,15 +122,21 @@ const linkGroups: {[key in LinkGroup]: {text: string, items: LinkItem[]}} = {
 
 export const defaultGroupLink = (linkGroup: LinkGroup) => groupLinks[linkGroup][0].link;
 
-export const linkGroup = (activeLinkGroup: LinkGroup | LinkGroup[]) => (
-  Object.keys(linkGroups).map((linkGroup: string) => ({
-    ...linkGroups[linkGroup as LinkGroup],
-    collapsible: true,
```

Comment:
Where did `collapsible` go? Is it simply implied by the existence of `collapsed`?

Reply:
My understanding is that it was removed from the Vitepress theme API. It relies on the presence of a `collapsed` boolean instead.

Docs:
https://vitepress.dev/reference/default-theme-sidebar#collapsible-sidebar-groups

Types now only have the `collapsed` option:

```typescript
type SidebarItem = {
  // …

  /**
   * If not specified, group is not collapsible.
   *
   * If `true`, group is collapsible and collapsed by default
   *
   * If `false`, group is collapsible but expanded by default
   */
  collapsed?: boolean
}
```